projects/storefrontlib/cms-structure/routing/default-routing-config.ts

```diff
@@ -81,4 +83,14 @@ export const defaultRoutesConfigFactory: () => RoutingConfig = () => {
       },
     },
   };
+
+  if (isNewAuthFlow) {
+    routingConfig.routing.routes['loginForm'] = {
+      paths: ['login'],
+      protected: false,
+      authFlow: true,
+    };
+  }
```

Comment:
`query:` could you help me understand what's the purpose of this? IS it related to the offline discussion we had yesterday?

Reply:
legacy: login route with path login
new: login route with path signin and loginForm with path login

we need 2 paths, but one cms page is enough (/login) so needn't update sampledata at all

Reply:
second thoughts - as we discussed, it indeed might be helpful, e.g. for redirection directly to the `/login` page.